### DISCUSSION TITLE
Compatibility with upcoming Blazor Server Side Rendering (SSR) ?

### DISCUSSION THREAD

**Discussion by timahrentlov** [bitfoundation/bitplatform#5977] [General]
Hi,

Looking at [MudBlazor](https://github.com/MudBlazor/MudBlazor/discussions/7430#discussioncomment-7273495) I noticed some concerns and problems with the upcoming SSR for Blazor in .NET 8.

Are there anything similar one should be aware of when it comes to bit BlazorUI ? For example, some requirement or "best practise" to put components in the layout layer that makes it problematic to run an overall SSR site with isles of more interactive (ex: WASM) content and controls?

**Comment by msynk** [comment]
@timahrentlov Thanks for raising this concern.
Our tests found no issues with using the bit BlazorUI components in the new BlazorWeb project template. 
However, we are in the middle of refactoring/rewriting our own project templates with the new approach introduced in .NET 8 so we can be more sure about the final state of this compatibility.

**Reply by timahrentlov** [reply]
Thanks @msynk

**Comment by yasmoradi** [comment]
I've prepared the first project template supporting new dotnet 8 structure
https://github.com/bitfoundation/bitplatform/pull/5991
@timahrentlov
